### DISCUSSION TITLE
Issue with Jupyter Theme

### DISCUSSION THREAD

**Discussion by winterbv3r** [visualpython/visualpython#202] [Q&A] [answered]
Hi there, I'm currently using the dark theme in Jupyter Notebook via Jupyter themes. However, I've noticed that since the Visual Python 2.3 version, the font color has been automatically changing to the light theme color, resulting in black letters on a black background. Could you please take a look and see if this can be fixed?

**Comment by minjk-bl** [comment]
Ok I'll take a look at this issue. Then, is there any screenshot or theme info that we can refer to?
It will be more helpful to us if you upload some more information about it! Thanks :)

**Reply by winterbv3r** [reply]
![vp_01](https://user-images.githubusercontent.com/122992457/224184452-84cebee5-9e02-482b-83cb-6b112fd212c0.png)
![vp_02](https://user-images.githubusercontent.com/122992457/224184482-5813b636-9091-442d-a25f-efcf22e7f6e7.png)

The first screenshot shows Visual Python version 2.2.12, while the second one shows the latest version of Visual Python. The Jupyter theme being used is 'jt -t oceans16 -f roboto -fs 10 -nfs 10 -T -N'.

**Reply by minjk-bl** [reply]
Thank you for giving us more information. 
We'll figure it out what is the problem, then it will be applied on the next version of Visual Python.
It will be updated very soon!

**Comment by minjk-bl** [comment, marked as answer]
Hello @winterbv3r ,
Today we updated new version of Visual Python v2.3.4.
This version will solve your issue :)
Thank you for using Visual Python.
